awx/api/serializers.py

```diff
@@ -773,6 +774,16 @@ def get_summary_fields(self, obj):
                 if val is not None:
                     summary_fields['source_workflow_job'][field] = val
 
+        if self.is_detail_view:
+            ancestor = obj.ancestor_job
+            if ancestor != obj:
+                summary_fields['ancestor_job']= {
+                    'id' : ancestor.id,
+                    'name': getattr(ancestor, 'name', None),
```

Comment:
I think `.name` is guaranteed to be on a `UnifiedJob`.

Reply:
Yep.